### PR TITLE
ncm-nfs: add server boolean and limit nfs service reload to export changes

### LINKED ISSUE #1029: ncm-nfs: why restart nfs when fstab is updated?
currently, on modifcations of fstab, a restart of the `nfs` service is triggered. why is this the case?
similar, is no exports are configured by `ncm-nfs`, why restart nfs at all?

this is flagged as a bug because it reports an error, yet nothing is wrong

### PR DESCRIPTION
**Pull request by stdweird**
Fixes #1029 
Replaces #1110

### CODE REVIEW

File: ncm-nfs/src/main/pan/components/nfs/schema.pan [hunk 1]
```diff
@@ -18,6 +18,10 @@ type structure_nfs_mounts = {
 
 type ${project.artifactId}_component = {
     include structure_component
+    @{Configure a NFS server. In particular relevant for missing exports attribute.
+      If true, missing exports forces an empty exports file and a NFs service reload.
```
Comment (ned21) [inline ncm-nfs/src/main/pan/components/nfs/schema.pan]: a NFs => an NFS

**Comment by jrha** [comment]
@stdweird could you update annotation as per @ned21's comment?

**Comment by stdweird** [comment]
@jrha done. but the travis check is wrong. via `fstab_simple` and `base`

**Comment by jrha** [comment]
It's not wrong, the template using it hasn't included it directly, consider it a warning and ignore if you want.

**Comment by stdweird** [comment]
@jrha jenkins fails on another component unittest. i added the commit here anyway

**Pull request merged**